asv_bench/benchmarks/benchmarks.py

```diff
@@ -401,3 +401,45 @@ def time_fillna(self, shape, limit, inplace):
             execute(self.df)
         else:
             execute(self.df.fillna(**kw))
+
+
+class BaseTimeValueCounts:
+    subset_params = {
+        "all": lambda df: df,
+        "half": lambda df: df.iloc[:, : len(df.columns) // 2],
+    }
+
+    def setup(self, shape, subset="all"):
+        self.df, _ = generate_dataframe(
+            ASV_USE_IMPL,
+            "int",
+            *shape,
+            RAND_LOW,
+            RAND_HIGH,
+            groupby_ncols=shape[1],
```

Comment:
Why are you using all columns?

Reply:
fixed, no groupby-like cols will be only cols from the `subset`